x-pack/plugins/lens/public/shared_components/axis/extent/axis_extent_settings.tsx

```diff
@@ -166,7 +165,7 @@ function MetricAxisBoundsControl({
           defaultMessage: 'Bounds',
         })}
         helpText={
```

Comment:
this is another non-blocker comment, but what if we remove this message from here and instead add the `tooltipContent` prop to the `Data` option?

<img width="500" alt="Screenshot 2024-04-26 at 16 08 34" src="https://github.com/elastic/kibana/assets/4283304/34a284d5-b447-42ec-a479-e898f7075b61">

Reply:
I like this a lot. Updated in 8c7a714.